.env.example

```diff
@@ -0,0 +1,10 @@
+MASTER_PASSWORD=master_password_changeme
```

Comment:
I'd say use a really easy to type password e.g. `secret` 
For somebody that copy/paste this example file, no need to edit thus more convenient

Reply:
Well security on your local machine? I don't understand. How is it secure? Or what will be the issues?
Remember, developers do not like to configure ... Just copy/paste and good to go: "sane defaults"